pyDeltaRCM/preprocessor.py

```diff
@@ -495,7 +495,9 @@ def _write_job_configs(self) -> None:
 
         # do not write any time information to file
         #   this list defines the time-related keywords
-        _time_list = ['timesteps', 'time', 'time_years']
+        _no_list = ['timesteps', 'time', 'time_years']
```

Comment:
so should we roll back these changes in the preprocessor, since we just ignore them in the init? 

Or should the preprocessor still screen out [parallel, matrix, set, ensemble], but not worry about these time ones? I think that [matrix, set, ensemble] are each popped from the dict in their processing steps if they are present, so we shouldn't need to worry about them. 

So, I think the best option is to probably roll back these changes and then add [parallel] to the list you've set up to ignore in init. Thoughts?

Reply:
> So, I think the best option is to probably roll back these changes and then add [parallel] to the list you've set up to ignore in init. Thoughts?

I agree, I think this is the best plan. I don't think matrix, set, or ensemble get written into individual job yamls.